src/js/base/model.js

```diff
@@ -36,13 +36,26 @@ export default Backbone.Model.extend(extend({
     // Model fetches default to aborting.
     const fetcher = Backbone.Model.prototype.fetch.call(this, extend({ abort: true }, options));
 
+    this._isFetching = true;
+
     // Resolve with entity if successful
     return fetcher.then(response => {
+      this._isFetching = false;
+
+      if (this._notifications) {
```

Comment:
no need for the if..  I think `each(this._notifications, this._handleMessage);` is fine here.

Reply:
Yep, not needed 👍